app.rb

```diff
@@ -57,4 +57,49 @@ def list_person_rentals(person_id)
       end
     end
   end
+
+  def create_person
+    puts 'Do you want to create a student(1) or a teacher(2)? [Input the number]:-'
+    option = gets.chomp
+    parent_permission = true
+    puts 'age:-'
+    age = gets.chomp.to_i
+    puts 'name:-'
+    name = gets.chomp
+    puts 'Has parent permission?[Y/N]'
+    val = gets.chomp
+    parent_permission = false if %w[N n NO no No].include?(val)
+
+    case option.to_i
+    when 1
+      create_student(age, name, parent_permission)
+    when 2
+      puts 'specialization:-'
+      specialization = gets.chomp
+      create_teacher(age, specialization, name, parent_permission)
+    else
+      puts 'Invalid input ):'
+    end
+  end
+
+  def handel_option(option)
+    case option
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      puts 'Enter person ID:-'
+      id = gets.chomp.to_i
+      list_person_rentals(id)
```

Comment:
- [ ] When I try to render the data based on the id, it throws error as attached to the image below, I think you are trying to load the date input too and that is where the issue arises. Kindly dig further.

![image](https://github.com/Koshao-sys/OOP-school-library/assets/86200356/b1fbfe8e-a164-4c43-bfa7-9baa62473ecc)

Reply:
I am just adding this, I forgot. The number 7 for exit is also not functioning as expected, rather, it rerenders the option. Kindly look into it.